### DISCUSSION TITLE
Can I set a default directory?

### DISCUSSION THREAD

**Discussion by CodeNinjaMatrix** [d2phap/ImageGlass#1030] [Q&A] [answered]
Hi,
Can I set a default dictionary when the software start up?

**Comment by d2phap** [comment, marked as answer]
What do you mean "dictionary"?

**Reply by CodeNinjaMatrix** [reply]
I meant "Directory", the location. Sorry for my bad English.

**Reply by d2phap** [reply]
IG does not directly support it, but I guess you can use the "Open last seen image" setting.

![image](https://user-images.githubusercontent.com/3154213/116341758-0add0000-a814-11eb-894d-ffbb2808ede3.png)

Steps are:
1. Check "Open last seen image" option in Settings > General > Start up.
2. Open installed directory of ImageGlass, create `igconfig.admin.xml` file.
3. Open configuration directory of ImageGlass, copy all content of `igconfig.xml` file to `igconfig.admin.xml`.
4. In `igconfig.admin.xml` file, delete all the settings, EXCEPT:
  - `IsOpenLastSeenImage`,  change its value to `True`, and
  - `LastSeenImagePath`, change its value to the image file of your directory.
5. Save your `igconfig.admin.xml` file.

You can read more about app configuration document at https://imageglass.org/docs/app-configs

**Reply by CodeNinjaMatrix** [reply]
After a little bit struggle with which *startup dir* for which config file, now it works!
Thank you so much for the step-by-step guide!
